frontend/model/contracts/shared/nativeNotification.js

```diff
@@ -9,20 +10,39 @@ export async function requestNotificationPermission (force: boolean = false): Pr
     (force && Notification.permission === 'granted')) {
     return null
   }
-
   try {
-    return await Notification.requestPermission()
+    const permission = await Notification.requestPermission()
+    if (permission === 'granted') {
+      sbp('state/vuex/commit', 'setNotificationGranted', true)
+    } else {
+      sbp('state/vuex/commit', 'setNotificationGranted', false)
```

Comment:
Could also slightly simplify here, with `sbp('state/vuex/commit', 'setNotificationGranted', permission === 'granted')`

Reply:
Yeah. You are the KING of Code Optimization. 😁

Reply:
@snowteamer See other discussion about this being removed completely: https://github.com/okTurtles/group-income/pull/1355#discussion_r926296100

Reply:
@greg Yea this makes my suggestion redundant